HelpSource/Classes/Object.schelp

```diff
@@ -692,8 +692,129 @@ code::
 method::dump
 
 Print a detailed low level representation of the receiver to the post window.
+Except for the link::Classes/List:: class, this method is not overridden in other classes. Any of the MetaClasses, Classes and Instances can be used by this method.
+
+code::
+Meta_Object.dump // the meta class of the class Object
+::
+code::
+Object.dump      // the class called Object
+::
+code::
+Object.new.dump  // an istance of the class Object
+::
+
+discussion::
+
+list::
+## The detailed low level format and information varies depending on the receiver.
+## Some instance objects, especially unique objects, return the class name and value (also low data if necessary) of the dumped object:
```

Comment:
I don't understand: "(also low data if necessary)". Maybe (also more data if necessary) ?

Reply:
Yes, your correction is better!
Thanks, done!